### PR TITLE
upgrade to uswds-jekyll 5

### PR DESCRIPTION
**Pull request by afeld**
Changes proposed in this pull request:
- Upgrades to the latest version of the theme
- Clears out some unnecessary cruft
- Adds the DAP subagency parameter

The site should look and act (basically) the same — just some changes under the hood.

### CODE REVIEW

**Comment by its-a-lisa-at-work** [comment]
@afeld can you please add someone as a reviewer (happy to as I am on OpsRotation) and set up a time to pair with someone on this since it seems like a hefty change.

**Comment by afeld** [comment]
It's hefty in that there were under-the-hood changes to work with the upgraded theme, but since it's a static site, basically just going for "is the site basically the same as it was before, in terms of layout, colors, etc?" Minor derivation is ok. Does that help clarify?

**Comment by afeld** [comment]
Regardless, happy to walk though the changes over video if you're curious!

File: Gemfile [hunk 1]
```diff
@@ -2,8 +2,8 @@ source 'https://rubygems.org'
 
 gem 'jekyll'
 gem 'jekyll-redirect-from'
-gem 'uswds-jekyll', '~> 4.0'
-gem 'kramdown-parser-gfm'
+gem 'uswds-jekyll', '~> 5.0'
+gem 'mini_racer' # for autoprefixer
```
Comment (afeld) [inline Gemfile:6]: Making this unnecessary in the future by adding to uswds-jekyll: https://github.com/18F/uswds-jekyll/pull/216

**Comment by afeld** [comment]
Re: https://github.com/18F/handbook/pull/2193#issuecomment-713675375 (meant for this repository), I think it's another one of those issues where it's broken on the Federalist preview URL because the site is deployed a different path (`baseurl`), but will work fine once live at https://join.tts.gsa.gov.

**Review by its-a-lisa-at-work — APPROVED** [review]
Assuming this site is meant to use a black color theme; right?

**Comment by afeld** [comment]
Yep; Not sure why it's customized in that way, but I was just trying to stay close to what was there before 🤷‍♂️

**Pull request merged**